product_variant_inactive/hooks.py

```diff
@@ -0,0 +1,17 @@
+from odoo import SUPERUSER_ID, api
+
+
+def uninstall_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    product_variant_action = env.ref(
```

Comment:
If you need to do this, it's because the technique used in the module is not the right one. Define another action and replace the reference of the action where needed instead.